src/content/docs/logs/enable-log-management-new-relic/configure-logs-context/configure-logs-context-apm-agents.mdx

```diff
@@ -21,158 +21,1190 @@ redirects:
+
+   ```
+   logging:
+     appenders:
+       - type: newrelic-file
+         # This format will be ignored by the newrelic-json layout, but used by the log-format layout.
+         logFormat: "%date{ISO8601} %c %-5p: %m trace.id=%mdc{trace.id} span.id=%mdc{span.id}%n"
+         layout: 
+           # type: newrelic-json
+           type: log-format
+   ```
+
+  </Collapser>
+
+  <Collapser
+    id="java-util-logging"
+    title="java.util.logging"
+  >
+
+To configure logs in context with the [`java.util.logging` package](https://docs.oracle.com/javase/7/docs/api/java/util/logging/package-summary.html):
+
+1. Update your project's dependencies to include the `java.util.logging` extension as applicable:
+
+   * Gradle: Add the following to your `build.gradle` file:
+
+     ```
+     dependencies {
+         compile("com.newrelic.logging:jul:2.0")
+     }
+     ```
+   * Maven: Add the following to your `pom.xml` file:
+
+     ```
+     <dependencies>
+         <dependency>
+             <groupId>com.newrelic.logging</groupId>
+             <artifactId>jul</artifactId>
+             <version>2.0</version>
+         </dependency>
+     </dependencies>
+     ```
+
+     If you can't edit these files, you can instead [add the jars directly to the application classpath](/docs/logs/enable-new-relic-logs/logs-context-java/java-adding-classpath/).
+     
+2. Check if your logging file's `handlers` property is set to something other than `NewRelicMemoryHandler`. Look for a line listing the root logger's handlers, like this:
+
+   ```
+   handlers = java.util.logging.FileHandler
+   ```
+3. Update your logging properties file to set the root logger's handler to `NewRelicMemoryHandler` so it intercepts messages destined for another handler:
+
+   ```
+   handlers = com.newrelic.logging.jul.NewRelicMemoryHandler
+   ```
+4. Configure the `NewRelicMemoryHandler` by setting the target to the handler that was previously assigned to the root logger, so it captures data New Relic needs on the thread the log message is coming from:
+
+   ```
+   com.newrelic.logging.jul.NewRelicMemoryHandler.target = java.util.logging.FileHandler
+   ```
+
+5. Use a `NewRelicFormatter` for the final handler. Update your logging properties file to set the `formatter` property like the following example. Make sure the handler where you set the formatter is the `target` handler from the previous step (`java.util.logging.FileHandler` in this example).
+
+   ```
+   java.util.logging.FileHandler.formatter = com.newrelic.logging.jul.NewRelicFormatter
+   ```
+
+     The New Relic log format is a tailored JSON format with specific fields in specific places that our log forwarder plugins and back end rely on. At this time, we don't support any customization of that format.
```

Comment:
> The New Relic log format is a tailored JSON format with specific fields in specific places that our log forwarder plugins and back end rely on

"The New Relic log format is JSON with telemetry metadata we use to correlate transactions and logs together. At this  time we don't support any customization of that format"